src/bucket/test/BucketIndexTests.cpp

```diff
@@ -272,6 +272,110 @@ class BucketIndexTest
         }
     }
 
+    void
+    testLoadContractKeysWithInsufficientReadBytes()
```

Comment:
This tests need refactoring to be less brittle, but currently it is passing on all index types except range index.